<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>624</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1044</width>
    <height>624</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1044</width>
    <height>624</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <widget class="QCustomPlot" name="graph" native="true">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(180, 179, 161);</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>270</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QGraphicsView" name="graphicsViewCoordinates">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>3</y>
      <width>271</width>
      <height>106</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80); border-radius: 12px; </string>
    </property>
   </widget>
   <widget class="QLabel" name="xLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>12</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(80,80,80); border: none;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>12</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(80,80,80); border: none;</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="endY">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>109</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50); color: white; border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="endX">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>109</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50); color: white; border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="endLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>109</width>
      <height>14</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(80,80,80); border: none;</string>
    </property>
    <property name="text">
     <string>End</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="startLabel">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>10</y>
      <width>108</width>
      <height>14</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: white; background-color: rgb(80,80,80); border: none;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="startX">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>30</y>
      <width>108</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50); color: white; border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="startY">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>70</y>
      <width>108</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50); color: white; border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>400</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <widget class="QGraphicsView" name="graphicsViewStepX">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>3</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80,80,80); border-radius: 12px; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="countStepX">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>35</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50); color: white; border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="countStepLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(80,80,80); border: none;</string>
    </property>
    <property name="text">
     <string>Count step</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="stepX">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>35</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50); color: white; border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="stepXLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(80,80,80); border: none;</string>
    </property>
    <property name="text">
     <string>Step X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="to_count_step">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	color: white;
	background-color: rgb(126, 44, 61); 
}
QPushButton:pressed {
	background-color: rgb(146, 64, 81); 
}
QPushButton:hover {
	border: 1px solid rgb(146, 64, 81); 
}</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_stepX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kefa</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 6px;
	color: white;
	background-color: rgb(126, 44, 61); 
}
QPushButton:pressed {
	background-color: rgb(146, 64, 81); 
}
QPushButton:hover {
	border: 1px solid rgb(146, 64, 81); 
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="createGraph">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kefa</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 6px;
	color: white;
	background-color: rgb(126, 44, 61); 
}
QPushButton:pressed {
	background-color: rgb(146, 64, 81); 
}
QPushButton:hover {
	border: 1px solid rgb(146, 64, 81); 
}</string>
   </property>
   <property name="text">
    <string>click me</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exprGraph">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>404</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color:#000;background-color:rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
